common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -282,7 +289,7 @@ div.video {
         }
       }
 
-      ul.vcr {
+      div.vcr {
```

Comment:
NIT: If we can avoid adding the "div" element style requirement here that would be great. Additionally, it wasn't clear to me from this what vcr stood for. It might be helpful to use more verbose class labeling here.